presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -355,9 +326,11 @@ public static String getColumnValueForCql(Row row, int position, CassandraType c
         }
     }
 
-    private static String objectToString(Object object, CassandraType elemType)
+    private static String objectToString(Object object, DataType dataType)
     {
-        switch (elemType) {
+        CassandraType cassandraType = toCassandraType(dataType.getName()).get();
```

Comment:
Can `toCassandraType` return empty here?

```suggestion
        CassandraType cassandraType = toCassandraType(dataType.getName())
           .orElseThrow( ....
```

Reply:
It should not be happened because the unsupported column is skipped from table layout. But let me update it.